k6_node/artifacts.mk

```diff
@@ -0,0 +1,5 @@
+K6_NODE_DIR=$(abspath services/k6_node)
+
+get.k6_node:
+	@echo "⇒ Creating keys for k6 node server and clients"
+	${K6_NODE_DIR}/generate_keys.sh > /dev/null
```

Comment:
And how will work with tests in the next step?
Here you have generated keys... What's next?

Reply:
Path to private key will be specified in variables for load test